INSTALL.md

```diff
@@ -7,116 +7,135 @@
 Installing OpenCoarrays
 =======================
 
-[![Download as PDF][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)
+[The following link is commented until it can be updated. To uncomment delete the backslashes, the ourtermost square brackets, and the trailing colon and hash mark]:#
+[\[\!\[Download as PDF]\][pdf img]](http://md2pdf.herokuapp.com/sourceryinstitute/opencoarrays/blob/master/INSTALL.pdf)]:#
```

Comment:
I reverted the commit that attempted to comment the PDF link.  When I tested it, I forgot that it points to the master branch and therefore would not reflect my changes on a separate branch so I mistakenly thought something had to be done to update the PDF.  Thanks for clarifying.